Makefile

```diff
@@ -117,6 +117,11 @@ _output/share/lima/lima-guestagent.Linux-riscv64:
 manpages: _output/bin/limactl$(exe)
 	@mkdir -p _output/share/man/man1
 	$< generate-man _output/share/man/man1
+	# Replace the output directory with the install prefix, in the manpages
+	sed -e 's|$(CURDIR)/_output|$(PREFIX)|' -i _output/share/man/man1/limactl.1
+	sed -e 's|$(CURDIR)/_output|$(PREFIX)|' -i _output/share/man/man1/limactl-sudoers.1
+	# Replace the home directory with a more generic reference, in the manpage
+	sed -e 's|$(HOME)|~|' -i _output/share/man/man1/limactl-sudoers.1
 
```

Comment:
Ideally these logics should be moved to the `generate-man` command, but it can be another PR in future

Reply:
I guess we can supply the output directory as a parameter or something